pkg/util/helperfunctions.go

```diff
@@ -32,6 +33,8 @@ var excludedNamespaces = []string{
 	"gatekeeper-system",
 	"gke-connect",
 	"gke-system",
+	"gmp-public",
+	"gmp-system",
```

Comment:
Vi kan godt ta med `gmp-test` også som brukes til å teste som en poc
```suggestion
	"gmp-system",
	"gmp-test",
```